docs/topics/development/error_pages.rst

```diff
@@ -0,0 +1,13 @@
+.. _error:
+
+===========
+Error Pages
+===========
+
+When running Django locally you get verbose error pages instead of the
+standard ones. To access the HTML for the standard error pages, you can
+access the urls::
+
+    /services/403.html
```

Comment:
I'd knock the `.html` off everywhere... and I'd also prefer just `/404`.

But cool!